velox/functions/sparksql/String.h

```diff
@@ -218,6 +218,29 @@ struct Sha2HexStringFunction {
   }
 };
 
+/// space function
+/// space(int) -> string
+/// Returns a string consisting of `n` spaces.
+template <typename T>
+struct SpaceFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<Varchar>& result,
+      const int32_t& count) {
+    if (count <= 0) {
+      result.setEmpty();
+      return;
+    }
+    result.resize(count);
+    std::memcpy(result.data(), generateSpaces(count).data(), count);
```

Comment:
I think we can use `std::memset`, no need to generate an extra string and then do the copy.

Reply:
> I think we can use `std::memset`, no need to generate an extra string and then do the copy.

updated. please review again